src/test/java/com/ontotext/kafka/error/LogErrorHandlerTest.java

```diff
@@ -0,0 +1,21 @@
+package com.ontotext.kafka.error;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.Test;
+
+public class LogErrorHandlerTest {
+
+	LogErrorHandler handler = new LogErrorHandler();
+
+/*	@Test
+	public void test() {
+		SinkRecord record = new SinkRecord("This is a failed record",
+				1,
```

Comment:
do you plan on adding unit tests for the log error handler? If not, remove class.